scripts/local-install.sh

```diff
@@ -53,14 +48,16 @@ COMPRESSED_PACKAGE="$(tail -1 /tmp/pack.out)"
 tar -xzf "${COMPRESSED_PACKAGE}"
 rm "${COMPRESSED_PACKAGE}"
 mv package "${PACKAGE_DIR}"
-PACKAGE_NAME=$(cat ./package.json | jq -r '.name')
+PACKAGE_NAME=$(node -p "require('./package.json').name")
```

Comment:
This removes the need for `jq`. Doing `npm pkg get name | sed -r 's/"//g'` would have also worked, but using node is a bit more straightforward and doesn't require any potentially breaking changes to future versions of npm.

Reply:
🙌